src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {person.name}
+    </h2>
+    <p className="Person__age">
+      {' '}
+      {person.age === undefined ? '' : `I am ${person.age}`}
+    </p>
+    <p className="Person__partner">
+      {person.partnerName}
+      {' '}
+      {(person.isMarried === true && person.sex === 'm') && (
+        'is my wife'
+      )}
```

Comment:
Use ternary here to show different messages
`          isMarried
            ? ${partnerName} is my ${partner}
            : 'I am not married'`